tools/ARIAtools/vrtmanager.py

```diff
@@ -364,4 +368,13 @@ def layerCheck(products, layers, nc_version, gacos_products, extract_or_ts):
                     '"-nc_version %s" constraint to filter older product '
                     'variants \n', layer_reject, nc_version)
 
-    return layers
+    # if specified, determine if computation of
+    # total tropospheric is possible
+    if tropo_corrections:
+        if not set(raider_tropo_layers).issubset(all_valid_layers):
+            log.warning('User-requested estimation of raider-derived total '
+                        'troposphere "-tc " is not possible as tropo '
```

Comment:
I have seen different uses of troposphere, toposphereTotal, Tc. Which one is it?